doc/src/equivalence.md

```diff
@@ -105,22 +129,21 @@ succeeds.
 
 If you want to check the equivalence of not just the runtime code, but also the
 creation code of two contracts, you can use the `--creation` flag.  For example
-these two contracts:
-
+the following two contracts compare equal when compared with `--create` flag.
+Let's set the first contract to [a.sol](code_examples/create1.sol):
 ```solidity
 contract C {
   uint private immutable NUMBER;
   constructor(uint a) {
     NUMBER = 2;
   }
   function stuff(uint b) public returns (uint256) {
-    unchecked{return 2+NUMBER+b;}
+    unchecked {return 2+NUMBER+b;}
   }
 }
 ```
```

Comment:
Isn't there a way to avoid duplicating the code in the file (`create1.sol`) and inlined in the Markdown?
Do we need the separate file?

Reply:
Not sure. I decided to also inline it into Markdown, so it's easy to see and in one place and it's an easy flow for the user.

Reply:
I fully agree with that. So, can we drop the solidity files?

Reply:
YES. Let me delete them all :) They are I think useless.